sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/EntityDescriptorFactory.scala

```diff
@@ -16,7 +16,7 @@
 
 package kalix.springsdk.impl
 
-import kalix.EntityMethodOptions.Generator
+import kalix.KeyGeneratorMethodOptions.Generator
```

Comment:
how was that not detected before?

Reply:
We change the protocol in workflows and adjusted this only on sdk workflow branch.